src/main/java/org/sunbird/extension/util/TransformationConstants.java

```diff
@@ -0,0 +1,27 @@
+package org.sunbird.extension.util;
+
+public final class TransformationConstants {
+
+  public static final String FROMFIELDNAME = "fromFieldName";
```

Comment:
Split words in constants with underscore.

e.g. FROMFIELDNAME to FROM_FIELD_NAME.